source/connect.txt

```diff
@@ -49,68 +49,77 @@ Connect
 
 When you open |compass-short|, an initial connection dialog appears:
 
-.. figure:: /images/compass/paste-connection-string.png
+.. figure:: /images/compass/new-connection.png
    :figwidth: 690px
-   :alt: Connection dialog for pasting your connection string
+   :alt: New connection dialog for pasting your connection string
 
-|compass-short| provides two methods to connect to your deployment,
-either providing your deployment connection string or filling in your
-deployment information in specific fields.
+You can connect your deployments by either:
 
-.. tabs::
+- Providing your connection string.
 
-   .. tab:: Paste Connection String
-      :tabid: paste-connection-string
+- Specifying :guilabel:`Advanced Connection Options`. 
 
-      When you provide a connection string, |compass-short| supports
-      most :manual:`Connection String Options
-      </reference/connection-string/#connection-string-options>`
-      supported by MongoDB.
+Advanced connection options allow you to specify authentication, 
```

Comment:
We should actually probably make this paragraph part of the second bullet.

Reply:
Done.